.github/workflows/test.yml

```diff
@@ -19,8 +23,8 @@ jobs:
   tests:
     strategy:
       matrix:
-        ruby-version: [3.1, 3.2, 3.3]
-        rails-version: [7.0, 7.1, 7.2, 8.0]
+        ruby-version: ["3.1", "3.2", "3.3", "3.4"]
+        rails-version: ["7.0", "7.1", "7.2", "8.0"]
```

Comment:
This change means we'll need to update the required checks to match the new check names

Reply:
Yep, just getting that sorted 👍

Reply:
Should probably remove Ruby 3.1 as a required check as well while you're asking for changes, as that goes EOL in just over a month

Reply:
Good shout 👍

Reply:
It should stay in the test matrix, just don't think it needs to be a required check